<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsPage</class>
 <widget class="QWidget" name="DetailsPage">
  <layout class="QVBoxLayout" name="layoutDetails">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="margin">
    <number>12</number>
   </property>

   <!-- Botón volver -->
   <item>
    <widget class="QToolButton" name="btnBack">
     <property name="text">
      <string>← Volver</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>

   <!-- Scroll principal -->
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>

     <widget class="QWidget" name="scrollContent">
      <layout class="QVBoxLayout" name="scrollLayout">
       <property name="spacing">
        <number>10</number>
       </property>

       <!-- Palabra principal -->
       <item>
        <widget class="QLabel" name="labelMainWord">
         <property name="styleSheet">
          <string>
font-size: 32px;
font-weight: bold;
          </string>
         </property>
         <property name="text">
          <string>—</string>
         </property>
        </widget>
       </item>

       <!-- Lecturas -->
       <item>
        <widget class="QLabel" name="labelReadings">
         <property name="styleSheet">
          <string>
font-size: 18px;
color: #555;
          </string>
         </property>
         <property name="text">
          <string>Lecturas</string>
         </property>
        </widget>
       </item>

       <!-- Separador -->
       <item>
        <widget class="QFrame" name="separator">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>

       <!-- Título sentidos -->
       <item>
        <widget class="QLabel" name="labelSensesTitle">
         <property name="text">
          <string>Significados</string>
         </property>
         <property name="styleSheet">
          <string>
font-size: 18px;
font-weight: 600;
          </string>
         </property>
        </widget>
       </item>

       <!-- Lista de sentidos -->
       <item>
        <widget class="QListWidget" name="listSenses"/>
       </item>

       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>

      </layout>
     </widget>
    </widget>
   </item>

    <item>
    <widget class="QPushButton" name="btnSrsAction">
    <property name="text">
    <string>Añadir al SRS</string>
    </property>
    </widget>
    </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
